src/Microsoft.OpenApi.OData.Reader/Edm/ODataOperationSegment.cs

```diff
@@ -112,6 +113,14 @@ public override string GetPathItemName(OpenApiConvertSettings settings, HashSet<
             return parameterNamesMapping;
         }
 
+        private string OperationName(IEdmOperation operation, OpenApiConvertSettings settings)
+        {
+            string selectedName = operation.FullName();
+            return settings.EnableUnqualifiedCall
+                ? settings.DefaultNamespace != null ? selectedName.RemoveDefaultNamespace(settings) : operation.Name
+                : selectedName;
```

Comment:
```suggestion
            return settings.EnableUnqualifiedCall
                ? selectedName.RemoveDefaultNamespace(settings) : operation.Name;
```

Reply:
This suggestion is now redundant following the suggestion in the issue discussion.